src/commands/sbom/license.ts

```diff
@@ -0,0 +1,96 @@
+import {DependencyLicense} from './types'
+
+// Get the license from a string. If the license is valid, we return it. Otherwise, we return undefined
+// List of licenses: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository
+const getLicenseFromString = (s: string): DependencyLicense | undefined => {
+  if (!s) {
+    return undefined
+  }
+
+  switch (s.toLowerCase()) {
+    case '0bsd':
+      return DependencyLicense.ZEROBSD
+    case 'apache-2.0':
+      return DependencyLicense.APACHE2
+    case 'bsd-2-clause':
+      return DependencyLicense.BSD2CLAUSE
+    case 'bsd-3-clause':
+      return DependencyLicense.BSD3CLAUSE
+    case 'bsl-1.0':
+      return DependencyLicense.BSL1
+    case 'isc':
+      return DependencyLicense.ISC
+    case 'mit':
+      return DependencyLicense.MIT
+    case 'unlicense':
+      return DependencyLicense.UNLICENSE
+    case 'zlib':
+      return DependencyLicense.ZLIB
+  }
+
+  console.debug(`license ${s} not recognized`)
+
+  return undefined
+}
+
+// Get all the licenses from a string. Sometimes, there are two licenses in one string
+// such as "MIT OR Apache-2.0". In this case, we return all the licenses in this condition.
+export const getLicensesFromString = (s: string): DependencyLicense[] => {
+  if (!s) {
+    return []
+  }
+  const licenses: DependencyLicense[] = []
+
+  if (s.indexOf('OR') !== -1) {
```

Comment:
You can use `includes()` here as well:
```suggestion
  if (s.includes('OR')) {
```